DayrSuryan/DSEthiop2.xml

```diff
@@ -230,121 +244,126 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+
                            <item xml:id="a5">
                               <locus target="#1r"/>
                               <desc type="OwnershipNote">Arabic Ownership note stating that this book belongs to <persName ref="PRS11940GabraK"/>.</desc>
                               <q xml:lang="ar">هذا الكتاب ملك
-                              <persName ref="PRS11940GabraK" role="owner"> 
+                              <persName ref="PRS11940GabraK" role="owner">
                                     <roleName type="title">القس</roleName> عبد المسيح الحبشي</persName>
                               </q>
                            </item>
-                        
+
                            <item xml:id="a6">
                               <locus target="#75ra"/>
-                              <desc type="OwnershipNote">Ownership note mentioning 
+                              <desc type="OwnershipNote">Ownership note mentioning
                               <persName ref="PRS11939TaklaH">
                                     <roleName type="title">ʾabbā</roleName> Takla Hāymānot</persName> of <placeName ref="INS0479DabraM">Dabra Māryām of ʾAbsādi</placeName>
                               </desc>
                               <q xml:lang="gez">
-                                 <hi rend="rubric">ዝመጽሐፍ፡ 
-                              ዘ<persName ref="PRS11939TaklaH" role="owner">
-                                       <roleName type="title">አባ፡ </roleName> ተክለ፡ ሃይማኖት፡</persName> ሀገሩ፡ 
+                                 <hi rend="rubric">ዝመጽሐፍ፡ ዘ<persName ref="PRS11939TaklaH" role="owner">
+                                       <roleName type="title">አባ፡ </roleName> ተክለ፡ ሃይማኖት፡</persName> ሀገሩ፡
                               <placeName ref="INS0479DabraM"> ደብረ፡ ማርያም፡</placeName> ዘአባ፡ አብሳዲ፡</hi>
                               </q>
                            </item>
-                        
+
                            <item xml:id="a7">
                               <locus target="#75ra"/>
                               <desc type="Condemnation">Beginning of a condemnation note</desc>
                               <q xml:lang="gez">ዘአቡነ፡ አብ<gap reason="illegible" unit="char" extent="2"/> ዘወሀብዎ፡ ዘሰረቆ፡ ወዘፈሐቆ፡</q>
                            </item>
-                        
+
                            <item xml:id="a8">
                               <locus from="75rb" to="76ra"/>
-                              <desc type="GuestText">Prayer to God, the following is the beginning:</desc> 
-                              <!--similar to LIT4859Litany-->
-                              <q xml:lang="gez">ወካእበ፡ ናስተበቈኦ፡ ለአምላከ፡ ምሕረት፡ ዘረዳእከ፡ ለአብርሃም፡ አመ፡ ይቀት፡ ሎሙ፡ ለነገስት፡ ዘአድሐንኮ፡ ለይስሐቅ፡
-                           እምነ፡ መጥቦሕት፡ ዘመራሕኮ፡ ለያዕቆብ፡ በውስተ፡ መንገድ፡</q>
-                              <!--the following is the explicit:-->
-                              <q xml:lang="gez">ፈኑ፡ እማዴ፡ ሰማይ፡ ወምድር፡ ባርክ፡ ማህበረነ፡ በኵሉ፡ ጊዜ፡ ወበኩሉ፡ ሰአት፡ በአሐዱ፡ ወልድከ፡ ዘቦቱ፡ ለከ፡ ምስሌሁ፡
```

Comment:
can we encode it (with using "gap")? I think I can only quote the additio in full, but here it would be too long, and here not really necessary. The entire text is in the ID record.

Reply:
Yes, this was always a problem for additions. Let's use `<gap reason="ellipsis"/>` for now, but I'll open an issue to add it to the guidelines.